Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/EcLite.asl

```diff
@@ -21,6 +21,9 @@ Scope (\_SB)
     Name (ECLA, 0) // Until the EcLite driver is ready, the control methods cannot access the operation region
     Name (BNUM, 0) // Number Of Batteries Present
 
+    Method (_STA, 0x0) {
+      Return(0x000F)

```

Comment:
It looks ECLT is always enabled. ECLE GNVS variable is not used to enable or disable ECLT?

Reply:
In BIOS the ECLE GNVS is use in BIOS setup menu so in SBL is always enabled. I will remove the ECLE GNVS in SBL.

Reply:
Will remove ECLE as this only valid for BIOS setup menu which not available in SBL.